--- debsigs.orig
+++ debsigs
@@ -27,6 +27,7 @@
 use List::Util qw(first);
 use IO::File;
 use POSIX ":sys_wait_h";
+use Text::ParseWords qw(parse_line);
 
 sub cmd_sign($);
 sub cmd_list($);
@@ -105,6 +106,10 @@
 
   my @cmdline = ("gpg", "--openpgp", "--detach-sign");
 
+  if ($gpgopts) {
+    push (@cmdline, parse_line('\\s+', 0, $gpgopts));
+  }
+
   if ($key) {
     push (@cmdline, "--default-key", $key);
   }
